app/views/extractorMetrics.scala.html

```diff
@@ -0,0 +1,46 @@
+@(extractorName: String, average: String, lastweeksubmitted: Int, lastmonthsubmitted: Int)(implicit user: Option[models.User])
+@import api.Permission
+
+@main("Extractor Details") {
+    <script src="@routes.Assets.at("javascripts/deleteUtils.js")" type="text/javascript"></script>
+    <script src="@routes.Assets.at("javascripts/extractors/extractors.js")" type="text/javascript"></script>
+    <style>
+            strong        { text-align: right; }
+            fieldset      { margin-top: 5px; }
+            .form-actions { margin-top: 15px; }
+    </style>
+
+    <ol class="breadcrumb">
+        <li><span class="glyphicon glyphicon-fullscreen"></span> <a href="@routes.Extractors.selectExtractors()">Extractor Catalog</a></li>
+        <li><span class="glyphicon glyphicon-search"></span> @extractorName</li>
+    </ol>
+
+    <div class="page-header"><h1>@extractorName Metrics Details</h1></div>
+
+    <div class="row">
+        <div class="col-sm-8">
+                <fieldset>
+
+                    <div class="row form-group">
+                        <strong class="col-sm-4" for="extractorNameField">Last 24 Hours Execution Average</strong>
+                        <span class="col-sm-8" for="extractorNameField">@average  Sec</span>
+                    </div>
+                        <!-- TODO: It would be nice to make this a mailto: link, but it doesn't seem trivial to scrape this out -->
+                    <div class="row form-group">
+                        <strong class="col-sm-4" for="extractorAuthorField">Last Week Submitted</strong>
+                        <span class="col-sm-8" for="extractorAuthorField">@lastweeksubmitted</span>
+                    </div>
+
+                    <div class="row form-group">
+                        <strong class="col-sm-4" for="extractorVersionField">Last Month Submitted</strong>
+                        <span class="col-sm-8" for="extractorVersionField">@lastmonthsubmitted</span>
+                    </div>
```

Comment:
`for="extractorXXField"` attributes are probably not necessary here, since it's not inside of a `<form>`.

Reply:
fixed